src/libostree/ostree-repo-pull.c

```diff
@@ -5784,6 +5784,13 @@ ostree_repo_remote_fetch_summary_with_options (OstreeRepo    *self,
   if (!ostree_repo_remote_get_gpg_verify_summary (self, name, &gpg_verify_summary, error))
     goto out;
 
+  if (gpg_verify_summary && summary == NULL)
+    {
+      g_set_error (error, G_IO_ERROR, G_IO_ERROR_NOT_FOUND,
+                   "No remote summary found (check that the configured URL is correct)");
```

Comment:
How about to be more consistent: "GPG verification enabled, but no summary found (check configured URL in remote config is correct)" ?

I guess technically, both this error and the next one should say "GPG *summary* verification enabled, ..." to be more precise.

Reply:
Sure